src/aw_watcher_input/config.py

```diff
@@ -0,0 +1,37 @@
+import argparse
+import sys
+
+from aw_core.config import load_config_toml
+
+default_config = """
+[aw-watcher-input]
+poll_time = 5
+
+[aw-watcher-input-testing]
+poll_time = 1
+""".strip()
+
```

Comment:
The configs are usually commented out by default.

Reply:
the `load_config_toml` function will write a commented out toml file based on the default its passed. you can see in [the aw-core repo](https://github.com/ActivityWatch/aw-core/blob/3b83ab542406349b3d5e601318289b77d23d906f/aw_core/config.py#L45)